core/src/main/java/nl/nn/adapterframework/webcontrol/api/Base.java

```diff
@@ -186,10 +186,10 @@ protected String resolveStringFromMap(Map<String, List<InputPart>> inputDataMap,
 		return result;
 	}
 
-	protected <T> T resolveTypeFromMap(Map<String, List<InputPart>> inputDataMap, String key, Class<T> clazz, T defaultValue) throws ApiException {
+	protected <T> T resolveTypeFromMap(MultipartBody inputDataMap, String key, Class<T> clazz, T defaultValue) throws ApiException {
 		try {
-			if(inputDataMap.get(key) != null) {
-				return inputDataMap.get(key).get(0).getBody(clazz, null);
+			if(inputDataMap.getAttachmentObject(key, String.class) != null) {
```

Comment:
Shouldn't this also be getAttachmentObject(key, clazz) ?
(Maybe you could store the object in a temporarily variable, so you don't have to fetch it twice.)